config/config_partial.go

```
@@ -443,6 +445,14 @@ func PartialParseConfig(ctx *ParsingContext, file *hclparse.File, includeFromChi
 			} else {
 				output.Dependencies = dependencies
 			}
+		case EngineBlock:
+			decoded := EngineConfig{}
+
+			err := file.Decode(&decoded, evalParsingContext)
+			if err != nil {
+				return nil, err
+			}
+			output.Engine = &decoded
```

golangci-lint (1.62.2)

assignments should only be cuddled with other assignments

(wsl)